tests/IssueStatusesTest.php

```diff
@@ -58,7 +58,30 @@ public function it_should_return_a_list_of_issue_statuses()
     }
 
     /** @test */
-    public function it_should_throw_an_failed_action_exception_when_client_receives_bad_request_while_getting_a_list_of_issue_statuses()
+    public function it_should_return_a_list_of_issue_statuses_for_an_issue_type()
```

Comment:
I believe the order is usually "happy path feat#1 - fail path feat#1 ... - happy path feat#2", etc